packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -499,8 +520,14 @@ export const start = async (zcf, privateArgs, baggage) => {
     },
   });
 
-  // @ts-expect-error types are correct. How to convince TS?
-  const scheduler = await makeScheduler(driver, timer, params, timerBrand);
+  const scheduler = await makeScheduler(
+    driver,
+    timer,
+    // @ts-expect-error types are correct. How to convince TS?
```

Comment:
glad to see the suppression more targeted. I'll take a look at this types after pushing this review.